frontend/src/components/contacts.tsx

```diff
@@ -1,136 +1,136 @@
-import React, { useState } from "react";
-import axios from "axios";
+import React, { useState } from 'react';
+import axios from 'axios';
 
-type ImportantDate = {
-  date: string;
-  event: string;
-};
+interface ImportantDate {
+  date: string
+  event: string
+}
 
-type Relationship = {
-  type: string;
-};
+interface Relationship {
+  type: string
+}
 
-export const Contacts = () => {
-  const [firstName, setFirstName] = useState<string>("");
-  const [lastName, setLastName] = useState<string>("");
-  const [relationships, setRelationships] = useState<Array<Relationship>>([]);
-  const [importantDates, setImportantDates] = useState<Array<ImportantDate>>(
+export const Contacts = (): JSX.Element => {
+  const [firstName, setFirstName] = useState<string>('');
+  const [lastName, setLastName] = useState<string>('');
+  const [relationships, setRelationships] = useState<Relationship[]>([]);
+  const [importantDates, setImportantDates] = useState<ImportantDate[]>(
     []
   );
-  const [eventDate, setEventDate] = useState<string>("");
-  const [eventType, setEventType] = useState<string>("");
-  const [relationshipType, setRelationshipType] = useState<string>("");
-  const [email, setEmail] = useState<string>("");
-  const [phoneNumber, setPhoneNumber] = useState<string>("");
-  const [notes, setNotes] = useState<string>("");
-  const [errorMessage, setErrorMessage] = useState<string>("");
-  const [showError, setShowError] = useState<boolean>(false);
+  const [eventDate, setEventDate] = useState<string>('');
+  const [eventType, setEventType] = useState<string>('');
+  const [relationshipType, setRelationshipType] = useState<string>('');
+  const [email, setEmail] = useState<string>('');
+  const [phoneNumber, setPhoneNumber] = useState<string>('');
+  const [notes, setNotes] = useState<string>('');
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [, setShowError] = useState<boolean>(false);
```

Comment:
what happened to `showError`? you call `setShowError` in a few places but it looks like the error itself doesn't currently exist?

Reply:
hm can you add a todo here - it def seems like a mistake here if we're setting showError in places but never consuming it

cc: @fahadakhan2

Reply:
sure, i messaged fahad to add a Todo. Fahad deleted this `contacts.tsx` file and created `contactsForm.tsx` in his own branch, so any changes here wouldn't transfer over when fahad merge his contact form branch.